src/logic/hydra.lean

```diff
@@ -79,10 +48,22 @@ open multiset prod
 
   The lemma `relation.cut_expand_iff` below converts between this convenient definition
   and the direct translation when `r` is irreflexive. -/
-def cut_expand (r : α → α → Prop) (s' s : multiset α) : Prop :=
+def cut_expand {α} (r : α → α → Prop) (s' s : multiset α) : Prop :=
 ∃ (t : multiset α) (a : α), (∀ a' ∈ t, r a' a) ∧ s' + {a} = s + t
 
-variable {r : α → α → Prop}
+variables {α : Type*} {r : α → α → Prop}
```

Comment:
Why moving the `variables` below?

Reply:
The variable α (and β) was used for the definition and properties of `relation.fibration` before it was moved to another file in #16720. Now `cut_expand` is the first declaration in the file, and I thought it's more compact to group `variables` together; even though α is still needed in `cut_expand`, it's just four characters to add it.

However, in order to minimize the diff, I have now restored the position of the variable declaration for α. Does it look good [now](https://github.com/leanprover-community/mathlib/pull/16772/files#diff-95ed0add602a6a7c7415fe5d698b86521513151e0b58d37a72c95924acbdad05L36-L55)?

Reply:
Yes thank you! The diff was quite misleading indeed.